plugins/vpp/ifplugin/interface_config.go

```diff
@@ -956,6 +965,18 @@ func (c *InterfaceConfigurator) canVxlanBeModifWithoutDelete(newConfig *intf.Int
 	return true
 }
 
+func (c *InterfaceConfigurator) canVmxNet3BeModifWithoutDelete(newConfig, oldConfig *intf.Interfaces_Interface_VmxNet3) bool {
+	if newConfig == nil || oldConfig == nil {
```

Comment:
This is not needed, since it's already done in `proto.Equal`: https://github.com/golang/protobuf/blob/master/proto/equal.go#L73-L75